Tests/SubprocessTests/SubprocessTests+Unix.swift

```diff
@@ -968,6 +968,35 @@ extension SubprocessUnixTests {
         }
         try FileManager.default.removeItem(at: testFilePath)
     }
+
+    @Test func testDoesNotInheritRandomFileDescriptorsByDefault() async throws {
+        // This tests makes sure POSIX_SPAWN_CLOEXEC_DEFAULT works on all platforms
+        let pipe = try FileDescriptor.ssp_pipe()
+        defer {
+            close(pipe.readEnd.rawValue)
+            close(pipe.writeEnd.rawValue)
+        }
+        let writeFd = pipe.writeEnd.rawValue
+        let result = try await Subprocess.run(
+            .path("/bin/bash"),
```

Comment:
I _think_ it's safer to assume a Unix system has `/bin/sh` instead of `/bin/bash`.

Reply:
Agree. FWIW, FreeBSD doesn't have bash installed by default, and it's located at /usr/local/bin/bash.